extensions/markdown-language-features/package.json

```diff
@@ -770,6 +770,8 @@
     "markdown-it-front-matter": "^0.2.4",
     "morphdom": "^2.6.1",
     "picomatch": "^2.3.1",
+    "punycode": "^2.3.1",
+    "tslib": "*",
     "vscode-languageclient": "^8.0.2",
```

Comment:
What happened here?

Reply:
The `tslib` is a leftover to satisfy peer-deps which we can remove with the `legacy-peer-deps` flags enabled.

As for `punycode`, it is to resolve the following error https://github.com/microsoft/vscode/pull/226927/commits/3b2cd961553df4e87172ca17b764c2741652b437. We can remove it once we update to `markdown-it >= 14.1.0`